docs/config/collection_signing.md

```diff
@@ -0,0 +1,336 @@
+# Enabling Collection Signing
+
+Galaxy Server can create, sync, receive as upload and verify and serve collection signatures.
+
+A signature is typically a GPG ASCII Armoured Detached artifact, in other words, a file
+e.g: `MANIFEST.json.asc` that is created by a GPG compatible script based on the `MANIFEST.json`
+file.
+
+The `MANIFEST.json` file already contains checksums to verify all the other artifacts inside a 
+collection so that is why the signature is created based on this file. 
+
+
+!!! tip "Enabling signing on pulp installer"
+    Pulp installer can also be configured to enable signing https://github.com/pulp/pulp_installer/tree/main/roles/galaxy_post_install#variables-for-the-signing-service
+    if you configures using pulp-installer you can skip the `Creating Signing Service` section of this page.
+
+
+## Creating Signatures
+
+Signatures can be manually created and then later uploaded to a galaxy server or the galaxy
+server can internally create a signature for each collection.
+
+### Signing Service
+
+A signing service is an object defined on the **pulp** backend that combines a GPG **key** and the
+absolute path an executable script.
+
+#### Creating a signing service
+
+To create a signing service on the Galaxy server it is needed to access the `django-admin` utility
+and execute:
+
+!!! info "important"
+    The command must run inside the Python Environment where galaxy_ng workers are installed.
+
+```bash title="Bash Command"
+django-admin add-signing-service \  #(1)
+    unique-name \  #(2)
+    /abs/path/to/script.sh \ #(3)
+    GPG_KEY_ID #(4)
+```
+
+1. A django management command.
+2. Name, the signing_service should get in the database.
+3. Absolute path to Shell script where the signing is created.
```

Comment:
Maybe either "where the signature is created" or "where the signing is performed"